src/woocommerce_plugin.js

```diff
@@ -116,6 +116,8 @@ export default class WooCommercePlugin {
     this._setElementValue(prefix + 'city', metaData.locality_name || '');
     this._setStateValue(prefix + 'state', metaData.state_territory);
     this._setElementValue(prefix + 'postcode', metaData.postcode);
+
+    this._dispatchEvent(document.body, 'update_checkout')
```

Comment:
the other lines have trailing semicolons, so we should probably keep that pattern (for now). 

Otherwise, ship it!